rpp30-boc-heartbleed/whimsical
# Add user registration info to remote db

## Description
Add user registration info to remote db, so it can be referenced and updated in the User Profile

## How to test
in /client/RegisterContainer.jsx, comment out line 130 (remote server url), uncomment line 129 (localhost url), load app, click 'register,' fill out and submit form, check console for data object returned from remote mongoDB in response with user info and document ID. Sign out from Dashboard. When returned to login screen, enter email and password used to register and click 'Login.' Should process and take you to Dashboard without having to enter info a second time.

## Notes
firebase account will not be created unless the user has been successfully registered in mongo.
no longer have to enter login info twice to succesfully log into app.

## Issue tracking
https://trello.com/c/4LFdKL3Z/44-add-user-registration-info-to-mongodb-user-profile-collection
https://trello.com/c/PjXu938y/43-fix-firebase-initalization
